owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -305,115 +309,105 @@ abstract class DrawerActivity : ToolbarActivity() {
-                                            userQuota.getRelative()
-                                        )
-                                    }
-                                    getAccountQuotaStatusText()?.apply {
-                                        visibility = View.VISIBLE
-                                        text = getString(R.string.drawer_nearing_quota)
-                                    }
-                                } else if (userQuota.state == UserQuotaState.CRITICAL || userQuota.state == UserQuotaState.EXCEEDED) { // Critical storage. Value over 90%
-                                    getAccountQuotaBar()?.apply {
-                                        isVisible = true
-                                        progress = userQuota.getRelative().toInt()
-                                        progressTintList = ColorStateList.valueOf(resources.getColor(R.color.quota_exceeded))
-                                    }
-                                    getAccountQuotaText()?.apply {
-                                        text = String.format(
-                                            getString(R.string.drawer_quota),
-                                            DisplayUtils.bytesToHumanReadable(userQuota.used, context, true),
-                                            DisplayUtils.bytesToHumanReadable(userQuota.getTotal(), context, true),
-                                            userQuota.getRelative()
-                                        )
-                                    }
-                                    getAccountQuotaStatusText()?.apply {
-                                        visibility = View.VISIBLE
-                                        text = getString(R.string.drawer_critical_quota)
-                                    }
-                                } else { // Normal storage. Value under 75%
-                                    getAccountQuotaBar()?.apply {
-                                        progress = userQuota.getRelative().toInt()
-                                        isVisible = true
-                                        progressTintList = ColorStateList.valueOf(resources.getColor(R.color.color_accent))
-                                    }
-                                    getAccountQuotaText()?.text = String.format(
-                                        getString(R.string.drawer_quota),
-                                        DisplayUtils.bytesToHumanReadable(userQuota.used, this, true),
-                                        DisplayUtils.bytesToHumanReadable(userQuota.getTotal(), this, true),
-                                        userQuota.getRelative()
-                                    )
-                                    getAccountQuotaStatusText()?.visibility = View.GONE
-                                }
-                            }
+        collectLatestLifecycleFlow(drawerViewModel.userQuota) { userQuota ->
```

Comment:
I did some changes in all these lines in the Detekt branch, I think just for the `MaxLineLength` rule. Try to keep them in the new place for these lines so that we don't duplicate work

Reply:
Done ✅